crypto/crypto.go

```diff
@@ -39,6 +39,8 @@ var (
 	secp256k1halfN = new(big.Int).Div(secp256k1N, big.NewInt(2))
 )
 
+var errInvalidPubkey = errors.New("invalid secp256k1 public key")
```

Comment:
I would make it exportable, so that whoever calls it  can compare the error, like:

```golang
err := MyFunctionThatCallsUnmarshalPubkeyAtAVeryNestedLevel()
if err == crypto.ErrInvalidPubkey {
  panic("crypto breach!")
} else {
 // recover the error
}
```

I'm aware you can still `String()`-compare, but it's nice to keep the type info as well.